tests/lib/DataMapper/ScheduleMapperTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GroupLife\Core\DataMapper;
+
+use GroupLife\Core\Schedule;
+use GroupLife\Core\Test\TestCaseWithDb;
+
+class ScheduleMapperTest extends TestCaseWithDb
+{
+    public function testInsert()
+    {
+        $schedule = new Schedule([
+            new Schedule\WeekdayRule('Monday', '10:00'),
+            new Schedule\CancelDayRule('2021-01-11', '10:00'),
+            new Schedule\CancelDayRule('2021-01-25', '10:00'),
+        ]);
+        $mapper = new ScheduleMapper(self::$db);
+        $mapper->insert($schedule);
+        $sqlQuery = self::$db->createQueryBuilder();
+        $sqlQuery
+            ->select('*')
+            ->from('schedule', 's')
+            ->join('s', 'schedule_rule', 'sr', 's.id = sr.schedule')
+            ->where('s.id = 1')
+            ->orderBy('s.id');
```

Comment:
FYI: it is not always necessary to use the query builder. Simple requests can be done with `fetch*` methods. See https://www.doctrine-project.org/projects/doctrine-dbal/en/2.12/reference/data-retrieval-and-manipulation.html